dokan/dokan.rc

```diff
@@ -5,7 +5,7 @@
 //
 // Generated from the TEXTINCLUDE 2 resource.
 //
-#include "windows.h"
+#include "rcinc.h"

```

Comment:
Why does winresrc.h not work? It works fine with dokan_fuse which is compiled both with MingW-w64 and the MS toolchain.

https://github.com/dokan-dev/dokany/blob/0c152df921c05951788e29a69cb8c8158764733f/dokan_fuse/src/dokanfuse.rc#L8

Reply:
Hmm, `winresrc.h` didn't come to my mind. I'll give it a try - it surely is much less complex as my solution.

Reply:
Ok, that works for me as well. Thank you for the hint!